config/clusters/2i2c-aws-us/showcase.values.yaml

```
@@ -48,6 +49,33 @@ basehub:
         name: quay.io/2i2c/researchdelight-image
         tag: "872f0c4578af"
       profileList:
+        - display_name: "NASA TOPS-T ScienceCore"
+          description: "Hub for collaborative work on 2i2c/MD's NASA TOPS-T ScienceCore Module"
```

If we mention this entry as "Hub", it could be implying a profile list entry is what a "jupyterhub" is.

Could it can be rephrased to avoid the ambiguity? Not sure what, but here was an idea

```suggestion
          description: "For collaborative work on 2i2c/MD's NASA TOPS-T ScienceCore Module"
```